### ISSUE TITLE
[Python 3.9] Mypy complains str has no attribute removesuffix

### ISSUE THREAD

**Issue by fniessink** [python/mypy#9548]
**Bug Report**

Mypy complains str has no attribute removesuffix with Python 3.9. 

**To Reproduce**

```shell
python3.9 -m venv venv
. venv/bin/activate
pip install mypy
echo '"123".removesuffix("3")' > test.py
mypy test.py
```

**Expected Output**

```console
Success: no issues found in 1 source file
```

**Actual Output**

```console
test.py:1: error: "str" has no attribute "removesuffix"
Found 1 error in 1 file (checked 1 source file)
```

**Your Environment**

- Mypy version used: 0.782
- Mypy command-line flags: none
- Mypy configuration options from `mypy.ini` (and other config files): none
- Python version used: 3.9.0
- Operating system and version: macOS Catalina, version 10.15.7

**Comment by hauntsaninja** [comment]
I added this in typeshed a couple months ago  / is fixed on master

**Comment by hauntsaninja** [comment]
Thank you for the report, though! :-)

**Comment by fniessink** [comment]
No problem. However, I'm not sure what's next. I checked the mypy documentation and FAQ but it isn't clear to me what I'm supposed to do to get mypy correctly checking Python 3.9 code?

**Comment by hauntsaninja** [comment]
You can install mypy from Github master or wait for the next release (#9290). Or you can continue to use mypy 0.782, download a copy of the latest [typeshed](https://github.com/python/typeshed/) and use `mypy --custom-typeshed-dir <location of downloaded typeshed>`.

(Based on your report this shouldn't apply, but if the Python you're using to *run* mypy is older, you'll need to pass in `--python-version 3.9`, as per https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-python-version)
